docs/overview/what_is__-__main_areas/con-application-connector.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Application Connector
+---
+
+The Application Connector (AC) is a custom, in-house built Kyma component that allows you to connect with external solutions. No matter if you want to integrate an on-premise or a cloud system, the integration process does not change, which allows to avoid any configuration or network-related problems.
+
+The external solution you connect to Kyma using the AC is represented as an Application. There is always a one-to-one relationship between a connected solution and an Application, which helps to ensure the highest level of security and separation. This means that you must create five separate Applications in your cluster to connect five different external solutions and use their APIs and event catalogs in Kyma.
+
+The Application Connector:
+
+- Manages lifecycles of Applications.
+- Establishes a secure connection and generates the client certificate used by the connected external solution.
+- Registers APIs and event catalogs of the connected external solution.
+- Delivers events from the connected external solution to Eventing.
+- Proxies calls sent from Kyma to external APIs registered by the connected external solution.
+- Allows to map an Application to a Kyma Namespace and use its registered APIs and event catalogs in the context of that Namespace.
+- Integrates the registered APIs and event catalogs with the Kyma Service Catalog.
+
+All of the AC components scale independently, which allows to adjust it to fit the needs of the implementation built using Kyma.
+
+
+## Supported APIs
```

Comment:
I'm on the fence if this section still answers the question "What is the Application Connector?" or if it's already a new topic. Not sure if this is where a user would look for that kind of information, maybe it suits better somewhere else.